packages/core/src/components/slider/slider.tsx

```diff
@@ -72,12 +72,18 @@ export class Slider extends CoreSlider<ISliderProps> {
         );
     }
 
-    protected handleTrackClick(event: React.MouseEvent<HTMLElement>) {
+    protected handleTrackClick(event: MouseEvent | React.MouseEvent<HTMLElement>) {
```

Comment:
why do we need to allow regular mouse events in addition to react's synthetic mouse events here (and in a few other places)?